Массивы1_5вариант/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Массивы1_5вариант
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите количество элементов массива");
+            var n = int.Parse(Console.ReadLine());
+            int[] array = new int[n];
+            Random rand = new Random();
+            for (int i = 0; i < array.Length; i++)
+            {
+                array[i] = rand.Next(-10, 11);
+                Console.WriteLine(array[i] + " ");
```

Comment:
Выводить на консоль элементы массива в первом задании не требуется. Тем более каждый элемент в отдельной строке. Для вывода нужно будет написать отдельный метод во втором задании.
```suggestion
                //Console.WriteLine(array[i] + " ");
```